opts/global.go

```diff
@@ -896,22 +896,49 @@ type Colors []string
 type Assets struct {
 	JSAssets  types.OrderedSet
 	CSSAssets types.OrderedSet
+
+	customizedJSAssets  types.OrderedSet
+	customizedCSSAssets types.OrderedSet
 }
 
-// InitAssets static assets initial options
+// InitAssets inits the static assets storage.
 func (opt *Assets) InitAssets() {
 	opt.JSAssets.Init("echarts.min.js")
 	opt.CSSAssets.Init()
+
+	opt.customizedJSAssets.Init()
+	opt.customizedCSSAssets.Init()
+}
+
+// AddCustomizedJSAssets adds the customized javascript assets which will not be added the `host` prefix.
+func (opt *Assets) AddCustomizedJSAssets(assets ...string) {
+	for i := 0; i < len(assets); i++ {
```

Comment:
I prefer to use for-range personally🐶。(no need to judge every time)

Reply:
It works as expected and I think it's no need to follow the `individual-code-style`